uint16 DOF
string[] names
float64[] velocities
